quarkchain/evm/messages.py

```diff
@@ -557,14 +567,15 @@ def _apply_msg(ext, msg, code):
                 want=msg.value,
             )
             # TODO: Why return success if the transfer failed?
-            return 1, msg.gas, []
-
-    if msg.transfer_token_id != ext.default_state_token:
-        # TODODLL calling smart contract with non QKC transfer_token_id is not supported
-        return 1, msg.gas, []
+            return 0, msg.gas, []
```

Comment:
note this change may break the consensus. need to make sure it conforms to our current history by using the db check tools.

on the other hand, seems this should never happen because transfer failure should be caught during validation. nevertheless, to be safe, we should run this change against db check tools

Reply:
Would you mind elaborating a bit on how to use the db check tools?